packages/dev/core/src/Materials/Node/Blocks/Input/inputBlock.ts

```diff
@@ -133,6 +132,7 @@ export class InputBlock extends NodeMaterialBlock {
                     case "world1":
                     case "world2":
                     case "world3":
+                    case "tangent":

```

Comment:
Is this change not a breaking change (switching the tangent from vec3 to vec4)? I guess some node materials can break because of that?

Reply:
I added the allow vector3 on the other ones and ensures deserialization was working for the ones saved with vec3

Reply:
Yes, but what if a node material is using a tangent attribute which is not directly connected to an input block but is doing some intermediate computing? It's probably not very common, but maybe adding a line in the "Breaking changes" section that now **tangent** is a vec4 would be appropriate.